tests/unit/test_sql_sensor.py

```diff
@@ -110,8 +110,8 @@ def tearDownClass(cls) -> None:
 
 
 class ClickHouseLegacySqlSensorTestCase(unittest.TestCase):
-    @mock.patch('airflow.sensors.sql.SqlSensor.poke')
-    @mock.patch('airflow.sensors.sql.SqlSensor._get_hook', create=True)
+    @mock.patch('airflow.providers.common.sql.sensors.sql.SqlSensor.poke')
+    @mock.patch('airflow.providers.common.sql.sensors.sql.SqlSensor._get_hook', create=True)
```

Comment:
This is a destructive change which fails for earlier versions of Airflow. When was `sensors.sql` moved to `providers.common.sql`? At which Airflow version?

Reply:
[Airflow 2.4.0 (2022-09-19)](https://airflow.apache.org/docs/apache-airflow/stable/release_notes.html?highlight=airflow+sensors+sql#id178) 

DBApiHook and SQLSensor have moved [(#24836)](https://github.com/apache/airflow/pull/24836)
DBApiHook and SQLSensor have been moved to the apache-airflow-providers-common-sql provider.